doc/getting_started.rst

```diff
@@ -1 +1,23 @@
 .. include:: ../README.rst
+
```

Comment:
I'm adding this extra information to close #90, since Sphinx -Gallery has its namespace